client/src/App.js

```diff
@@ -0,0 +1,48 @@
+import "./App.css";
+import io from "socket.io-client";
+import { useState } from "react";
+import Chat from "./Chat";
+
+const socket = io.connect("http://localhost:3001");
+
+function App() {
+  const [username, setUsername] = useState("");
+  const [room, setRoom] = useState("");
+  const [showChat, setShowChat] = useState(false);
+
+  const joinRoom = () => {
+    console.log(username, room);
```

Comment:
no console logs should be pushed

Reply:
Apologies for the silly mistake. Will be trying to avoid in future.